cloudformation/elk-stack.json

```diff
@@ -112,6 +116,134 @@
 
     "Resources": {
 
+        "MediaServiceKinesisSenderRole": {
+            "Type": "AWS::IAM::Role",
+            "Properties": {
+                "Path": "/",
+                "AssumeRolePolicyDocument": {
+                    "Statement": [
+                        {
+                            "Action": "sts:AssumeRole",
+                            "Effect": "Allow",
+                            "Principal": {
+                                "AWS": { "Fn::Join": [ "", [ "arn:aws:iam::", { "Ref": "MediaServiceAccountId" }, ":root" ] ] }
+                            }
+                        }
+                    ]
+                }
+            }
+        },
+
+        "KCLDynamoDBTable" : {
+            "Type" : "AWS::DynamoDB::Table",
+            "Properties" : {
+                "AttributeDefinitions" : [
+                    {
+                        "AttributeName" : "leaseKey",
+                        "AttributeType" : "S"
+                    }
+                ],
+                "KeySchema" : [
+                    {
+                        "AttributeName" : "leaseKey",
+                        "KeyType" : "HASH"
+                    }
+                ],
+                "ProvisionedThroughput" : {
+                    "ReadCapacityUnits" : "10",
+                    "WriteCapacityUnits" : "5"
+                }
+            }
+        },
```

Comment:
@sihil the stream name must be provided manually to senders. There isn't a mechanism for automatic discovery.